test/functional/apps/visualize/_tsvb_chart.ts

```diff
@@ -86,7 +89,7 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
         const labelString = await PageObjects.visualBuilder.getGaugeLabel();
         expect(labelString).to.be('Count');
         const gaugeCount = await PageObjects.visualBuilder.getGaugeCount();
-        expect(gaugeCount).to.be('156');
+        expect(gaugeCount).to.be('13,830');
```

Comment:
Why this needs to change?

Reply:
I think it because with that changes by default we use `EntireTimeRange` instead of `LastValue`. 13,830 was calculated for the whole timeperiod

Reply:
Yes, but if you see above it sets the timerange to Last Value 
`await PageObjects.visualBuilder.setMetricsDataTimerangeMode('Last value');`